src/mlpack/methods/ann/layer/isru.hpp

```diff
@@ -0,0 +1,195 @@
+/**
+ * @file isru.hpp
+ * @author Prince Gupta
+ *
+ * Definition of ISRU (Inverse Square Root Unit) activation function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_ISRU_HPP
+#define MLPACK_METHODS_ANN_LAYER_ISRU_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artifical Neural Network. */ {
+
+/**
+ * The ISRU (Inverse Square Root Unit) activation function, defined by
```

Comment:
Do you think we should cite: https://arxiv.org/abs/1710.09967 here, similar to:

https://github.com/mlpack/mlpack/blob/f68fc14a77fbab24d586092ceac2294f62564b18/src/mlpack/methods/ann/layer/highway.hpp#L39-L49